<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/sq2/outdoor_dkan/2018-12-11-13-37-56.bag -s 100"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find msg_conversion)/rviz_config/odometry_reset_origin.rviz"/>

	<!-- MAIN -->
	<node pkg="msg_conversion" type="odometry_reset_origin" name="odometry_reset_origin" output="screen">
		<param name="parent_frame_id" type="string" value="/odom"/>
		<param name="child_frame_id" type="string" value="/odom/reset"/>
	</node>
</launch>
